addons/nametags/CfgVehicles.hpp

```diff
@@ -12,7 +12,35 @@ class CfgVehicles {
         isGlobal = 1;
         icon = QUOTE(PATHTOF(UI\Icon_Module_NameTags_ca.paa));
         class Arguments {
-            class PlayerNamesViewDistance {
+            class showPlayerNames {
+                displayName = "$STR_ACE_NameTags_ShowPlayerNames";
+                description = "$STR_ACE_NameTags_ShowPlayerNames_Desc";
+                typeName = "SCALAR";
```

Comment:
Already fixed, will push with the rest of the stuff, thanks a lot @Glowbal for telling me this!

#BIS' fault